src/js/media-controller.js

```diff
@@ -673,6 +693,8 @@ const monitorForMediaStateReceivers = (
   const unsubscribe = () => {
     // Unregister any Media State Receiver descendants (including ourselves)
     traverseForMediaStateReceivers(rootNode, unregisterMediaStateReceiver);
+    // Make sure we remove all slotchange event listeners
+    slotAndChangeHandlers.forEach(([slotEl, eventHandler]) => slotEl.removeEventListener(eventHandler));
```

Comment:
I think `removeEventListener` is missing the event name as a first argument here